DI-435/merge_VCF_AF.sh

```
@@ -12,6 +12,33 @@ genome=$2
 job="${DX_JOB_ID}"
 num_proc="$(grep -c ^processor /proc/cpuinfo)"
 
+exec 1> >(tee -a -i "/home/dnanexus/dx_stdout")
+exec 2> >(tee -a -i "/home/dnanexus/dx_stderr")
+
+# Prefixes all lines of commands written to stdout with datetime
+PS4='\000[$(date)]\011'
+export TZ=Europe/London
+
+# set frequency of instance usage in logs to 10 seconds
+sudo kill $(ps aux | grep pcp-dstat | head -n1 | awk '{print $2}')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)